src/ProjectForm.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import { Card, CardContent, Stack, TextField, Button } from '@mui/material';
+import Paper from '@mui/material/Paper';
+import { styled } from '@mui/material/styles';
+import { DesktopDatePicker, LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment';
+import moment from 'moment';
+
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  color: theme.palette.text.secondary,
+  alignItems: 'center',
+  width: '100%',
+}));
+
+function ProjectForm() {
+  const [name, setName] = useState('');
+  const [projectName, setProjectName] = useState('');
+  const [date, setDate] = useState(moment());
+  const [description, setDescription] = useState('');
+
+  function handleChange(e) {
+    switch (e.target.id) {
+      case 'name':
+        setName(e.target.value);
+
+        break;
+      case 'description':
+        setDescription(e.target.value);
+        break;
```

Comment:
I would add the `default` case as the `eslint` may complain. Try running through `eslint` setup for the project.